apps/nft-service/src/nft/validate.ts

```diff
@@ -1,5 +1,5 @@
 import { log } from '@thecointech/logging';
-import { verifyMessage } from 'ethers/utils';
```

Comment:
The TS version revert seems to have issues with importing directly from sub-folders.